lib/cli.js

```diff
@@ -19,6 +19,7 @@ if (cmd == 'deploy') {
   // TODO serverless deploy
 }
 
+console.log(process.env.RERUN_LIMIT)
```

Comment:
do we want to log it here or was that for testing?

Reply:
That was for testing 🤦